### DISCUSSION TITLE
save button broken

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#2220] [General]
Probably already being looked at, but just in case. Clicking the save button on the "From Text" tab throws an error and doesn't save the image.

Commit hash c45ac963

11:28:33-168837 ERROR    gradio call: TypeError
╭───────────────────────────────────────── Traceback (most recent call last) ──────────────────────────────────────────╮
│ /home/afbag/sdnext/modules/call_queue.py:58 in f                                                                     │
│                                                                                                                      │
│   57 │   │   │   │   pr.enable()                                                                                     │
│ ❱ 58 │   │   │   res = func(*args, **kwargs)                                                                         │
│   59 │   │   │   if res is None:                                                                                     │
│                                                                                                                      │
│ /home/afbag/sdnext/modules/ui_common.py:113 in save_files                                                            │
│                                                                                                                      │
│   112 │   │   │   if shared.opts.use_save_to_dirs_for_ui:                                                            │
│ ❱ 113 │   │   │   │   namegen = modules.images.FilenameGenerator(p, seed=p.all_seeds[i], promp                       │
│   114 │   │   │   │   dirname = namegen.apply(shared.opts.directories_filename_pattern or "[pr                       │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: FilenameGenerator.__init__() missing 1 required positional argument: 'index'

**Comment by vladmandic** [comment]
sorry about that, fixed now.
but why is this in discussions when its clearly an issue?

**Reply by afbagwell** [reply]
Apologies. Since it had been awhile prior to the last commit I figured it was a good chance of already being on your radar.
